cypress/integration/fetch_items.js

```diff
@@ -1,6 +1,6 @@
 describe("Show a list", function() {
   it('Changes url to represent current view when clicked', function() {
-		cy.visit('/')
+		cy.visit('/add')
```

Comment:
I'm a little confused by this - if we're testing to show the list, should we go to the list page and not the add page?

Reply:
Not sure what was going on here. If I made that edit it was by accident. I changed it to list, and it passes.